src/api/query.rs

```diff
@@ -123,6 +174,11 @@ pub enum Address {
     BakingRewardAccount,
     /// Virtual finalization reward account.
     FinalizationRewardAccount,
+    /// Virtual foundation accrue account.
+    FoundationAccrueAccount,
+    /// Virtual pool accrue account. Baker ID of None denotes the accrue account
+    /// of the passive pool.
+    PoolAccrueAccount(Option<BakerId>),
```

Comment:
I recommend using a dedicated enum. 

```rust
pub enum Pool {
    Baker(BakerId),
    Passive,
}
```

It is easier to understand when it is used, without having to look up this documentation to figure out the meaning of `None`.

Reply:
The SDK's `get_pool_status` accepts this type and it's only used there so I'd rather keep it like this. But agree that it's better so maybe the SDK should use that type?